readthedocs/rtd_tests/tests/test_project_forms.py

```diff
@@ -261,20 +261,45 @@ def test_list_only_active_versions_on_default_version(self):
             {'latest', 'public-1', 'public-2', 'private', 'protected'},
         )
 
-    def test_list_all_versions_on_default_branch(self):
+    def test_list_only_non_auto_generated_versions_on_default_branch(self):
+        user_created_stable_version = get(
+            Version,
+            project=self.project,
+            slug='stable',
+            active=True,
+            privacy_level=PUBLIC,
+            identifier='ab96cbff71a8f40a4240aaf9d12e6c10',
+            verbose_name='stable',
```

Comment:
Can we move the test for `stable` after the _normal_ tests? Also, we need a test for a branch named `latest`. Maybe we can just put this in another test case.

Reply:
@stsewd as these tests are for ProjectAdvancedForm I think we should put it here. And as you said we can just add a new test method inside this test case for `stable` and `latest`. And should I put only the assertions for `commit_name` inside the new test method?

Reply:
@stsewd Updated the Tests. Let me know if this will work for you. :)